org.eclipse.wildwebdeveloper/src/org/eclipse/wildwebdeveloper/InitializeLaunchConfigurations.java

```diff
@@ -51,11 +51,12 @@ public static String getNodeJsLocation() {
 
 		String res = which("node");
 		if (res == null) {
-			res = getDefaultNodePath();
+			if (Files.exists(Paths.get(getDefaultNodePath()))) {
+				res = getDefaultNodePath();
+			}
 		}
 
-		if (Files.exists(Paths.get(res))) {
-
+		if (res != null) {
```

Comment:
Can getDefaultNodePath return invalid location in some cases (ie node has moved or something like that? If so, then it could make sense to keep the check.

Reply:
Yes, but I didn't get rid of the check -- I just moved it up a bit so it's only run if ```which("node")``` returns null.

Reply:
Ok